DocumentService.API/Models/LoginRequest.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace DocumentService.API.Models;
+
+public class LoginRequest
```

Comment:
We don't need a login request for document service.
The idea is to setup a authentication token in the .env file. We will share this token with those who want to use our service.
The token should be of bearer token.

Reply:
Followed the approach of having a preset JWT_KEY.
All APIs need authentication token for it